src/flows/pipes/Notification/readOnly.tsx

```diff
@@ -0,0 +1,276 @@
+// Libraries
+import React, {FC, useContext, useMemo, lazy, Suspense} from 'react'
+import {parse} from '@influxdata/flux'
+import {
+  ComponentStatus,
+  Form,
+  FlexBox,
+  Input,
+  InputType,
+  Icon,
+  IconFont,
+  ComponentSize,
+  Panel,
+  AlignItems,
+  Dropdown,
+  InfluxColors,
+  TechnoSpinner,
+  SpinnerContainer,
+} from '@influxdata/clockface'
+import Threshold from 'src/flows/pipes/Notification/Threshold'
+
+import {PipeContext} from 'src/flows/context/pipe'
+import {FlowQueryContext} from 'src/flows/context/flow.query'
+import {remove} from 'src/shared/contexts/query'
+import {DEFAULT_ENDPOINTS} from 'src/flows/pipes/Notification/Endpoints'
+const NotificationMonacoEditor = lazy(() =>
+  import('src/flows/pipes/Notification/NotificationMonacoEditor')
+)
+
+// Types
+import {RemoteDataState} from 'src/types'
+import {PipeProp} from 'src/types/flows'
+
+// Styles
+import 'src/flows/pipes/Notification/styles.scss'
+
+const ReadOnly: FC<PipeProp> = ({Context}) => {
+  const {id, data, results, loading} = useContext(PipeContext)
+  const {simplify, getPanelQueries} = useContext(FlowQueryContext)
+  let intervalError = ''
+  let offsetError = ''
+
+  if (!data.interval) {
```

Comment:
why is there data validation on a readonly component? the user can't fix it, so lets not tell them about it